arrow-buffer/src/builder/offset.rs

```diff
@@ -0,0 +1,238 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::ops::{Add, Sub};
+
+use crate::{ArrowNativeType, OffsetBuffer, ScalarBuffer};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct OffsetsBuilder<O: ArrowNativeType> {
+    offsets: Vec<O>,
+}
+
+/// builder for [`OffsetBuffer`]
```

Comment:
```suggestion
/// Builder for [`OffsetBuffer`]
```

Can we capitalize the first word of doc comments please